yarn-error.log

```diff
@@ -0,0 +1,36 @@
+Arguments: 
+  /usr/local/Cellar/node@10/10.16.3/bin/node /usr/local/Cellar/yarn/1.19.1/libexec/bin/yarn.js add @cognite/eslint-plugin-cognite --save-dev
+
+PATH: 
+  /usr/local/opt/openjdk/bin:/usr/local/opt/node@10/bin:/Users/larsmoastuen/.cargo/bin:/Users/larsmoastuen/.config/nvm/12.18.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/share/dotnet:~/.dotnet/tools:/usr/local/opt/openjdk/bin:/Users/larsmoastuen/.cargo/bin:/usr/local/opt/node@10/bin
+
+Yarn version: 
+  1.19.1
+
+Node version: 
+  10.16.3
+
+Platform: 
+  darwin x64
+
+Trace: 
```

Comment:
@larsmoa I added some .gitignore for yarn stuff in your branch, since your changes reminded me about it, but I added it for viewer/. This file should be deleted I guess. 

I see you wanted to add some eslint stuff here, but doesn't look like you finished on that.

Reply:
Experiment that I didn't follow up on - oops